routes/views/beerProfile.js

```diff
@@ -0,0 +1,27 @@
+var keystone = require('keystone');
+
+exports = module.exports = function(req, res) {
+    
+    var view = new keystone.View(req,res),
+        locals = res.locals;
+
+    locals.section = 'beerProfile';
+    locals.data = {
+        allBeers: []
+    };
+
+    view.on('init', function(next) {
+
+        var q = keystone.list('Beers').model.find().sort('-beerName')
```

Comment:
You should take the slug in here and find one- look at Mongoose's findOne method.